src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -522,9 +516,6 @@ hg.comp hf
 lemma norm_noninc_of_isometry (hf : isometry f) : f.norm_noninc :=
 λ v, le_of_eq $ norm_eq_of_isometry hf v
 
-lemma bound_by_one_of_isometry (hf : isometry f) : f.bound_by 1 :=
```

Comment:
Should this instead be replaced by `norm_le_one_of_isometry`?

Reply:
There is `norm_noninc_of_isometry` that combined with `norm_noninc_iff_norm_le_one` gives what you want.

It depends if we prefer to state lemmas about `f.norm_noninc` or about `∥f∥ ≤ 1` (or both of course).